apps/judicial-system/backend/src/app/modules/defendant/test/internalDefendantController/updateDefendant.spec.ts

```
@@ -0,0 +1,77 @@
+    givenWhenThen = async () => {
+      const then = {} as Then
+
+      await internalDefendantController
+        .updateDefendant(
+          caseId,
+          defendantNationalId,
+          { id: caseId, defendants: [defendant] } as Case,
+          defendant,
+          update,
+        )
+        .then((result) => (then.result = result))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)